doc/user/data/sql_funcs.yml

```diff
@@ -362,11 +362,11 @@
     - signature: 'digest(data: text, type: text) -> bytea'
       desctription: >-
         Computes a binary hash of the given text `data` using the specified `type` algorithm.
-        Currently, only the `md5` algorithm is supported.
+        Supported hash algorithms include: `md5`, `sha224`, `sha256`, `sha384`, `sha512`.
     - signature: 'digest(data: bytea, type: text) -> bytea'
       desctription: >-
         Computes a binary hash of the given bytea `data` using the specified `type` algorithm.
-        Currently, only the `md5` algorithm is supported.
+        Supported hash algorithms include: `md5`, `sha224`, `sha256`, `sha384`, `sha512`.
```

Comment:
Maybe just say "The hash algorithms that are supported are the same as for the `text` variant of this function" so that we don't need to duplicate? Up to you!